code-challenges.js

```diff
@@ -31,24 +52,60 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const bpMeasure = (num) => {
+if (num < 212) {
+return `Tempurature ${num} is below boiling point`
+}
+else if (num > 212) {
+    return `Temperature ${num} is above boiling point`
+}
+else {
+    (num === 212) 
+    return `Temperature ${num} is at boiling point`
+}
+}
+
+console.log(bpMeasure(temperature1))
+console.log(bpMeasure(temperature2))
+console.log(bpMeasure(temperature3))
+// Outcome: 
+// Tempurature 42 is below boiling point
+// Temperature 350 is above boiling point
+// Temperature 212 is at boiling point
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// Input: The two provided arrays
+// Output: A number which is the length of the two arrays combined
+// Process: Use the + operator to combine the two arrays and use .length to measure the length
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+console.log(padres1984WorldSeriesRuns.length + (padres1998WorldSeriesRuns.length))
+// Outcome: 9
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+// Input: The given variable
+// Output: The given variable reversed
+// Process: To use the .split, .reverse and .join built in methods to get desired output
 
 const currentCohort = "Charlie 2023"
 // Expected output: "3202 eilrahC"
 
+console.log(currentCohort.split('').reverse().join('').toString())
```

Comment:
Nice work! Note: don't need `toString()` here because `.join('')` is already converting the array to a string! Also, try to make your pseudo code more descriptive!